### DISCUSSION TITLE
Platform UI returns None / Log: Manifest file not found

### DISCUSSION THREAD

**Discussion by johanneshk** [inventree/InvenTree#7695] [Q&A] [answered]
Hey,
I installed Inventree using the installer today on Ubuntu 20. Works fine so far but now I wanted to access the new platform UI so I can use the template editor but the browser just displays 'None'.

```
Jul 19 14:41:05 inventree inventree-web-1.service[2501]: Manifest file not found
Jul 19 14:41:05 inventree inventree-web-1.service[2501]: 192.168.178.156 - - [19/Jul/2024:14:41:05 +0000] "GET /platform HTTP/1.0" 200 401 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:125.0) Gecko/20100101 Firefox/125.0"
```

![Screenshot_20240719_164351](https://github.com/user-attachments/assets/3002f891-6b7f-4e65-b509-6675d49bbeb9)

Is there some configuration I missed?

**Comment by matmair** [comment, marked as answer]
A fix for this is in stable and will be fixed in the next release - https://github.com/inventree/InvenTree/pull/7655

**Reply by ia2115** [reply]
isnt this fixed in 2025? i have still this issue newest . revision

**Comment by ia2115** [comment]
2025-01-07T11:46:01.692020Z [error    ] Manifest file not found        [inventree] ip=myip request_id=5a340921-b2df-47cb-87e7-609763a37071 user_id=None
2025-01-07 11:46:01,692 ERROR {'event': 'Manifest file not found', 'request_id': '5a340921-b2df-47cb-87e7-609763a37071', 'ip': 'myip', 'user_id': None, 'timestamp': '2025-01-07T11:46:01.692020Z', 'logger': 'inventree', 'level': 'error'}   

inventree is started as dev.server -a localip:8000 

 git pull
remote: Enumerating objects: 268, done.
remote: Counting objects: 100% (257/257), done.
remote: Compressing objects: 100% (70/70), done.
remote: Total 268 (delta 113), reused 240 (delta 107), pack-reused 11 (from 1)
Receiving objects: 100% (268/268), 750.85 KiB | 25.03 MiB/s, done.
Resolving deltas: 100% (113/113), completed with 39 local objects.
From https://github.com/inventree/inventree
   ce617b779..3a62bdd27  master                                                       -> origin/master
   4c6e3490c..8576fbbad  0.17.x                                                       -> origin/0.17.x
 * [new branch]          dependabot/npm_and_yarn/src/frontend/dependencies-92423fe3ce -> origin/dependabot/npm_and_yarn/src/frontend/dependencies-92423fe3ce
 + 09f798b2c...d345e54c3 l10_crowdin                                                  -> origin/l10_crowdin  (forced update)
Updating ce617b779..3a62bdd27
Fast-forward
 src/backend/InvenTree/order/api.py                                           | 58 +++++++++++++++++++++++++++-------------------------------
 src/backend/InvenTree/order/test_api.py                                      | 10 +++++++++-
 src/backend/InvenTree/plugin/registry.py                                     |  4 +---
 src/backend/InvenTree/stock/models.py                                        |  6 ++----
 src/frontend/src/pages/Index/Scan.tsx                                        |  2 +-
 src/frontend/src/pages/Index/Settings/AdminCenter/CurrencyManagmentPanel.tsx |  1 +
 6 files changed, 41 insertions(+), 40 deletions(-)

just run 07/01/25 13:15

**Reply by matmair** [reply]
thank you for reusing this old issue with a different install way and release target. this will surely be helpfull
